app/source/components/graph/graph.jsx

```diff
@@ -30,22 +30,26 @@ const themes = {
   }
 };
 
-const Graph = ({ className, data, theme }) => {
+const Graph = ({ className, data, showTooltips, theme }) => {
   if (!Object.keys(data).length) {
     return null;
   }
 
   return (
     <div className={cn('graph', className, theme.className)}>
-      <Line height={40} {...graphUtils.getChartProps(data, theme.canvas)} />
+      <Line
+        height={40}
+        {...graphUtils.getChartProps(data, theme.canvas, showTooltips)}
+      />
     </div>
   );
 };
 
 Graph.propTypes = {
   className: PropTypes.string,
   data: PropTypes.object,
-  theme: PropTypes.oneOf(Object.values(themes))
+  theme: PropTypes.oneOf(Object.values(themes)),
+  showTooltips: PropTypes.bool
```

Comment:
`show` is a verb. Verbs should be executable `verb()`. I would recommend `shouldShowTooltips` or something like it.

Reply:
Sure, but is `showTooltips` of type `Boolean` really ambiguous?